pkg/kubelet/kubeletconfig/checkpoint/store/fsstore_test.go

```diff
@@ -306,13 +306,12 @@ func TestFsStoreAssignedModified(t *testing.T) {
 		t.Fatalf("could not change timestamps, error: %v", err)
 	}
 
-	// for now we hope that the system won't truncate the time to a less precise unit,
-	// if this test fails on certain systems that may be the reason.
+	// compare the times with RFC3339 format because certain systems truncate the time to a less precise unit.
 	modTime, err := store.AssignedModified()
 	if err != nil {
 		t.Fatalf("unable to determine modification time of assigned config source, error: %v", err)
 	}
-	if !now.Equal(modTime) {
+	if now.Format(time.RFC3339) != modTime.Format(time.RFC3339) {
```

Comment:
Would it be better to just pick a time for "now" that has the least-common-precision across systems?

Reply:
Well, indeed. I will fix it. Thanks!